datus-spark/scripts/init_tpch_data.py

```
@@ -0,0 +1,214 @@
+
+    print("\nInserting TPC-H data...")
+    row_counts = [5, 25, 10, 15, 5]
+    for i, data in enumerate(TPCH_DATA):
+        conn.execute_ddl(data)
+        print(f"  Inserted {row_counts[i]} rows into {TPCH_TABLES[i]}")
+
+    # Verify
+    print("\nVerifying data...")
+    for i, table in enumerate(TPCH_TABLES):
+        result = conn.execute(
+            {"sql_query": f"SELECT COUNT(*) AS cnt FROM `default`.`{table}`"},
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)